cf/crypto.m4

```diff
@@ -135,6 +135,17 @@ if test "$crypto_lib" = "unknown"; then
 
 fi
 
+AC_ARG_WITH(pkcs11-module,
```

Comment:
See other mail, it could be done but doing it properly would require a lot of thought and probably some API changes that would make hcrypto a superset (well, intersecting set) of libcrypto.

Sent from my iPhone

> On 29 Nov 2015, at 11:58, Nico Williams <notifications@github.com> wrote:
> 
> In cf/crypto.m4:
> 
> > @@ -135,6 +135,17 @@ if test "$crypto_lib" = "unknown"; then
> >  
> >  fi
> >  
> > +AC_ARG_WITH(pkcs11-module,
> Ahh! Got it. Thanks. Also, there's no place in sight where we can store any configuration.
> 
> —
> Reply to this email directly or view it on GitHub.
>